### PR TITLE
Capella test vectors

### PR DESCRIPTION
**Pull request by hwwhww**
- Fix `capella/fork.md`: since `ExecutionPayloadHeader` is changed in Capella, we need to implement new `BeaconState.latest_execution_payload_header` object.
- Add Capella to test vector generators
- Also add `eip4844/ssz_static` together... other test generators are WIP.

I've generated it successfully locally.

### CODE REVIEW

**Comment by Joumax11** [comment]
> * Solución `capella/fork.md`: dado `ExecutionPayloadHeader`que se cambia en Capella, necesitamos implementar un nuevo `BeaconState.latest_execution_payload_header`objeto.
> * Agregue Capella para probar generadores de vectores
> * También agregue `eip4844/ssz_static`juntos... otros generadores de prueba son WIP.

............. M......

File: tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py [hunk 1]
```diff
@@ -195,6 +204,11 @@ def random_block_bellatrix(spec, state, signed_blocks, scenario_state):
     return block
 
 
+def random_block_capella(spec, state, signed_blocks, scenario_state):
+    block = random_block_bellatrix(spec, state, signed_blocks, scenario_state)
```
Comment (djrtwo) [inline tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py:208]: `build_random_block_from_state_for_next_block()` (which is used under the hood) does not add `BLSToExecutionChange` operations. So we should conditionally add in that function or just add some random ones here (I think doing here is cleaner but then the original functoin name is slight off, it should maybe be `build_random_phase0_block..`

File: tests/generators/fork_choice/main.py [hunk 2]
```diff
@@ -17,11 +17,12 @@
         'on_merge_block',
     ]}
     bellatrix_mods = combine_mods(_new_bellatrix_mods, altair_mods)
-
+    capella_mods = bellatrix_mods
```
Comment (djrtwo) [inline tests/generators/fork_choice/main.py:20]:
```suggestion
    capella_mods = bellatrix_mods  # No additional Capella specific fork choice tests
```

File: tests/generators/genesis/main.py [hunk 3]
```diff
@@ -8,18 +8,19 @@
...
-    # we have new unconditional lines in `initialize_beacon_state_from_eth1` and we want to test it
     altair_mods = phase_0_mods
 
+    # we have new unconditional lines in `initialize_beacon_state_from_eth1` and we want to test it
     _new_bellatrix_mods = {key: 'eth2spec.test.bellatrix.genesis.test_' + key for key in [
         'initialization',
     ]}
     bellatrix_mods = combine_mods(_new_bellatrix_mods, altair_mods)
-
+    capella_mods = bellatrix_mods
```
Comment (djrtwo) [inline tests/generators/genesis/main.py:18]:
```suggestion
    capella_mods = bellatrix_mods  # No additional Capella specific genesis tests
```

**Review by djrtwo — COMMENTED** [review]
Looks good! quick note about getting BLSChange operations inside of random tests. otherwise, good to go

**Comment by Where-2** [comment]
Can you tell me am I getting stolen from


Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py (hunk 1 above)
Comment (hwwhww) [reply tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py:209]: added in https://github.com/ethereum/consensus-specs/pull/3040/commits/0e2c3d89e020f43e0c2af1c246ec772ec54ee250

**Pull request merged**